src/pages/InstructorRecordings/InstructorRecordings.jsx

```diff
@@ -213,7 +192,7 @@ const InstructorRecordings = () => {
   });
 
   useEffect(() => {
-    fetchRecordings();
+    fetchRecordings(setRecordings);
```

Comment:
We should do something like:
`fetchQuizzes().then((res) => setQuizzes(res.data.quizzes));`
`fetchRecordings` should only fetch the recordings.